config/elasticsearch.yml

```diff
@@ -55,6 +55,8 @@ xpack.security.transport.ssl.enabled: ${XPACK_SECURITY_TRANSPORT_SSL_ENABLE}
 xpack.security.http.ssl.enabled: ${XPACK_SECURITY_HTTP_SSL_ENABLE}
 xpack.security.transport.ssl.verification_mode: certificate
 xpack.security.audit.enabled: ${XPACK_SECURITY_AUDIT_ENABLE}
+xpack.security.audit.index.events.exclude: [ "${XPACK_AUDIT_INDEX_EVENTS_EXCLUDE}" ]
```

Comment:
Does it expect an array or single string with comma separated event names? If the former then you'd have to ensure values are interpolated as expected so it doesn't break configuration.